config/fastly/whitelisted_params.yml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
These were copied over from our current Fastly setup. If you see params that we are no longer using, please call them out and I'll remove them in this PR.

Reply:
I'd leave a comment here at the top of the file to remind people of keeping this sorted unless we change the code to not rely on it.

Reply:
The code already doesn't rely on this being sorted - this file can be in any order. I just ordered it when I created the file because it's easier for people to scan to see what's in Fastly. I sort it in the task:
```ruby
FILE_PARAMS = YAML.load_file("config/fastly/whitelisted_params.yml").sort.freeze`
```